libraries/AP_Proximity/AP_Proximity.cpp

```diff
@@ -65,6 +65,15 @@ const AP_Param::GroupInfo AP_Proximity::var_info[] = {
     // 19 was _MIN
     // 20 was _MAX
 
+    // @Param: _MIN_OBS_ALT
+    // @DisplayName: Proximity lowest altitude.
+    // @Description: Minimum altitude below which proximity should not work.
+    // @Units: m
+    // @Range: 0 10
+    // @User: Advanced
+    AP_GROUPINFO("_MIN_OBS_ALT", 25, AP_Proximity, _min_obs_alt_m, 1.0f),
```

Comment:
This needs to be set to Copter-only param. Look above at how "IGN_GND" is added. "AP_GROUPINFO_FRAME"

Reply:
I think we should maybe name the parameter _ALT_MIN

Reply:
I missed this.... it is done now.